TESTING.md

```diff
@@ -91,18 +91,22 @@ This will instruct all JVMs (including any that run cli tools such as creating t
 -   `tests.class` is a class-filtering shell-like glob pattern
```

Comment:
I think this test case filtering patterns are not applicable anymore, right?

Reply:
You're right, thank you for the comment!
I deleted it and substituted with an example